openquake/hazardlib/shakemap/parsers.py

```diff
@@ -205,6 +209,89 @@ def get_array_usgs_xml(kind, grid_url, uncertainty_url=None):
             'USGS xml grid file could not be found at %s' % grid_url) from e
 
 
+def convert_to_oq_rupture(rup_json):
+    """
+    Convert USGS json into an hazardlib rupture
+    """
+    ftype = rup_json['features'][0]['geometry']['type']
+    assert ftype == 'MultiPolygon', ftype
+    multicoords = rup_json['features'][0]['geometry']['coordinates'][0]
+    hyp_depth = rup_json['metadata']['depth']
+    rake = rup_json['metadata'].get('rake', 0)
+    trt = 'Active Shallow Crust' if hyp_depth < 50 else 'Subduction IntraSlab'
+    Mw = rup_json['metadata']['mag']
+    rup = get_multiplanar(multicoords, Mw, rake, trt)
+    return rup
+
+
+# Namespace
+F64 = numpy.float64
+NAMESPACE = 'http://openquake.org/xmlns/nrml/0.4'
+NRML05 = 'http://openquake.org/xmlns/nrml/0.5'
+GML_NAMESPACE = 'http://www.opengis.net/gml'
+SERIALIZE_NS_MAP = {None: NAMESPACE, 'gml': GML_NAMESPACE}
+PARSE_NS_MAP = {'nrml': NAMESPACE, 'gml': GML_NAMESPACE}
+
+
+# Write node
```

Comment:
This function looks useless, a copy of `openquake.hazardlib.nrml.write`. Please check.

Reply:
I removed the duplicate, thanks